flybywiresim/aircraft
# feat(a380x/fma): disconnect ap for ldg message

Fixes #[issue_no]

## Summary of Changes

Adds the "DISCONNECT AP FOR LDG" FMA message on PFD.
## Screenshots (if necessary)


![image](https://github.com/user-attachments/assets/5487835e-04cf-4532-9c53-a1e4e463c87d)
![image](https://github.com/user-attachments/assets/4e82b55f-bbe7-49e1-8eb2-14368b0b617d)

## References





## Additional context



Discord username (if different from GitHub):
bruno_pt99
## Testing instructions

1) Go to or below 150FT RA with  one AP engaged with any of the following modes:
2) DES, OP DES, V/S or FPA (with 0 or negative value). The DISCONNECT AP FOR LDG message should appear on the PFD.
3) Fail the 3 RA systems through the EFB and make sure you have a landing airport inserted on the FMS.
4) Go to an altitude which is less or equal to the destination landing elevation +150FT with any of the modes mentioned in nr2.  DISCONNECT AP FOR LDG message should appear once again (this one is easier to test with altimeter setting manipulation rather than having to fly close to the ground 😅 )
5) Disconnect autopilot after the FMA message is visible and it should disappear.
6) Fly an ILS approach and let the autopilot Autoland, the message should not appear.

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
